index.html

```diff
@@ -1389,6 +1389,34 @@ <h2>A project for first-time contributions</h2>
         </div>
          <!-- ________ You-Know-Who card END ________  -->
 
+
+          <!-- ________ Parth Dwivedi card START ________  -->
+        <div class="card">
+          <p class="name">Parth Dwivedi</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com" target="_blank">@_Parth_Dwivedi_</a>
```

Comment:
Hi, please note that you should change the link of your profile account, so it's not the website of main page.
You should change: 
- [ ] twitter link
-  [ ] github link

```suggestion
            <a href="https://www.twitter.com/{your username}" target="_blank">@_Parth_Dwivedi_</a>
```